response_operations_ui/templates/reporting-units.html

```diff
@@ -40,18 +40,24 @@ <h1 class="jupiter">Reporting units</h1>
                         <th scope="col" class="table--header--cell">
                             Name
                         </th>
+                        <th scope="col" class="table--header--cell">
+                            Trading as
```

Comment:
Do we still want this to appear if there is no trading as?

Reply:
Yeah should just show up empty in the list of business returned.